ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -7,13 +7,16 @@
 using ntbs_service.Services;
 using ntbs_service.Models;
 using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+using ntbs_service.Models.Validations;
+using ntbs_service.Pages;
 using ntbs_service.Models.Enums;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.AspNetCore.WebUtilities;
 
 namespace ntbs_service.Pages_Search
 {
-    public class IndexModel : PageModel
+    public class IndexModel : ValidationModel
     {
         public INotificationService service;
         public int PageIndex;

```

Comment:
I think page variables should also be refactored into a separate class, which could be general purpose for any future paging needs.

Reply:
PageIndex is being deleted here - left it in unused by accident. The majority of the paging logic is kept within PaginationList.cs, the SetPaginationDetails() method could arguably be generalised and kept somewhere else but inclined to leave it until we do need it's functionality again